compiler/rustc_target/src/spec/aarch64_unknown_trusty.rs

```diff
@@ -0,0 +1,27 @@
+// Trusty OS target for AArch64.
+
+use super::{PanicStrategy, RelroLevel, Target, TargetOptions};
+
+pub fn target() -> Target {
+    Target {
+        llvm_target: "aarch64-unknown-linux-musl".into(),
```

Comment:
Trusty is Little Kernel based, right? Not Linux based.

Reply:
Yeah you're right. I've switched the llvm targets to `aarch64-unknown-unknown-musl` and `armv7-unknown-unknown-gnueabi`.